torch/distributions/normal.py

```diff
@@ -24,6 +24,10 @@ class Normal(Distribution):
     """
 
     def __init__(self, mean, std):
+        if mean.size() != std.size():
```

Comment:
If we want to support numbers then we'll also have to wrap them in Tensors or Variables. As @apaszke has mentioned, it would be nice to have a helper `broadcast_all(*params)` that could correctly deal with mismatched type (`float` vs `Tensor`) and shape (`(1,4)` vs `(5,1)`).

Reply:
Yeah, I had not realized that numbers are also supported. Will add a helper utility to take care of this.